content/post/writing-a-book-remotely.md

```diff
@@ -0,0 +1,581 @@
+
+We started with the free plan of ConvertKit and recently migrated to MailLite. The MailerLite pricing and plan structure was better for our needs.
+
+Some other tools/services that we used:
+- Grammarly for fundamental proofreading and typos
+- Writing the eBook in their editor of choice. I personally used IntelliJ IDEA most of the time.  I tweaked the editor settings to wrap the lines so that I can write more naturally without having to scroll endlessly horizontally or buy a curved monitor.
+- Fiverr for designing creatives, the logo, and the eBook cover
+- GitHub for collaboration and review
+- Google Sheet to keep track of our costs
+- Slack for internal and external (with the community) communication
+
+### Find a Schedule to Keep Momentum
+
+What helped us to keep the writing flow going was a fixed schedule. We met every two weeks for one hour and did a status update before planning the next steps.
+
+What worked for us was:
+- A Trello board with four columns: backlog, in progress, to review, done
+- Meetings via Zoom
+- Document writing standards within the repository
+- Use GitHub Actions for basic automation
+
+What personally worked for me was taking the first hour every morning to work on this project.
+
+### Automate What Can Be Automated
+
+As we all had a software developer background, we tried to automate everything that we could. We could have also automated the eBook publishing using Leanpub's API.
+
+GitHub Actions makes such automations almost effortless.
+
+Adding links for further resources is both a curse and a blessing. It allows the interested reader to dive deeper into a topic. However, the links disappear over time and may no longer be accessible as the content might have moved, or the site may have been shut down.
+
+We automated the hyperlink verification with a small GitHub Action and an npm package that checks all links (also internal manuscript links) on each commit:
+
+```
+name: Check for broken links
+on: [push, pull_request]
+jobs:
+  broken-link-check:
+    runs-on: ubuntu-latest
+    name: Check for broken links in all Markdown files
+    steps:
+      - name: Check out source code
+        uses: actions/checkout@v3
+
+      - name: Set up Node 16
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          cache: 'npm'
+
+      - name: Install NPM packages
+        run: npm install
+
+      - name: Check for broken links
+        run: npm run check-links
+```
+
+### Reach Out for Advertisement
+
+Start talking about your eBook on podcasts, conferences, and tweet about it as soon as possible.
+
+We reached out to an AWS developer advocate during the inception phase to find possible ways of collaboration.
+
+AWS has sponsored us with some gift vouchers so that we could run the infrastructure (almost) for free. We could even sponsor a two-part article on the AWS Open Source blog to gain some traction for our project:
```

Comment:
```suggestion
AWS has sponsored us with some gift vouchers so that we could run the infrastructure (almost) for free. We even were able to contribute a two-part article on the AWS Open Source blog to gain some traction for our project:
```